gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Sat Aug 28 16:51:29 CST 2021
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```

Comment:
Is there an important reason to change to a package that contains code? Binary packages can save time in third-party automated build tools.

Reply:
It has sources and documentations and It will provide IDE with Gradle API/DSL documentation. Good for debugging.

Reply:
To facilitate collaboration, in future versions I will use packages that contain the code.